src/components/List/index.js

```diff
@@ -1,4 +1,5 @@
-/* @flow */
+// TODO: enable flow, issue for components that use props.children
+// https://github.com/facebook/flow/issues/1820
```

Comment:
Children is used other places, shouldn't be any. Try searching for it in some of our other components.